### DISCUSSION TITLE
宿舍生活

### DISCUSSION THREAD

**Discussion by lai1008** [1101-GS4719-C/Python-Community-GS4719-C#2246] [General]
想問問看大家現在已經住進宿舍大概快兩個月了，有沒有甚麼最不習慣的事，或是跟住家裡比起來最大的不同?

**Comment by DarsooK** [comment]
住進宿舍對很多人來說確實是一個很大的轉變，特別是和在家裡的生活相比，有不少需要適應的地方。我認為這兩個變化是最不習慣的：

1. 隱私空間減少
宿舍通常是多人合住的空間，隱私難免比在家裡少了很多。想要有自己的私人空間，可能需要和室友協調、安排好共同生活的方式。這對於習慣獨處或喜歡安靜的同學來說，會是個挑戰。

2. 生活作息的不同
每個人的作息時間和習慣可能都不一樣，有人早起，有人晚睡。尤其在不同科系或課程的安排下，大家的時間安排各異，這也可能導致生活上需要互相調整適應。有人可能會覺得這樣很有趣，但對有些人來說可能是難以適應的地方。

**Comment by Tchang323** [comment]
我現在的室友從我大一就一起住到現在，已經快四年了XD，都沒有甚麼不習慣的地方，不過我覺得在宿舍會讓我比較認真，做事情效率比較高，因為在家裡常常會耍廢、吸貓，或是陪媽媽追劇。

**Comment by NiCL100** [comment]
不習慣的應該就是跟室友的睡覺時間不同。我以前十二點睡就覺得很晚了，沒想到室友都兩、三點才睡，結果我現在也變成十二點、一點多才睡的人，早一點上床反而睡不著。室友還會跟我說：「你老人喔？那麼早睡。」不過他們人很好，看到我在睡覺都會放輕動作，讓我覺得有點不好意思。有趣的是，常常有人兩、三點還很大聲的喧嘩，然後就會有人去敲他們的門，更會有人在宿舍群組上罵，被講的也不甘示弱，便回嗆說：「怕吵就不要住宿啊！」。雖然我覺得人應該要和平相處，但其實看到這樣的互動，也覺得蠻有趣的，當然，我不表認同。

**Comment by gigitilto** [comment]
宿舍與家裡主要的差異在於公共與私密，家裡的隱私性較高，比較不需顧慮與提防別人，而在宿舍若沒有顧慮與防人之心，容易吃虧。但也不是沒有好處，宿舍有設施可以使用，相較家中可以活動的空間多了許多，所以我覺得住宿對我來說是比較好的。

**Comment by christaaah** [comment]
我覺得住宿舍也很舒服，可能是遇到比較合得來的室友，不過我也有看過住一年就馬上外宿的，再加上聽我外校朋友說過她遇到的室友的一些事情讓我覺得室友是住宿舍舒不舒服最大的原因。

**Reply by yuan3x2** [reply]
就是說，要好好珍惜

**Comment by Boying12** [comment]
主要是沒有空間做一些自己的事，像是盡情地玩遊戲、居家健身，還有廁所比較麻煩，但個人認為住宿舍對學業還是幫助不小，宿舍有讀書氛圍的話可以有督促效果，還可以請教同學，當然，沒有的話，那還是出去住比較自在。

**Comment by qiu0109** [comment]
大一的時候跟室友作息時間差很多，我都是半夜回宿舍的那種，每次回宿舍都很害怕吵到室友，而且在宿舍沒辦法讓電子設備的聲音外放讓我有點困擾。但也有一些好處像是可以跟室友培養感情，有些重要訊息考試資訊忘記了都可以問室友，當你睡過頭室友還會把你叫醒，如果你的室友都是卷姐的話還可以問作業。總之是在外住宿舍室友真的很重要。

**Comment by wcyeee** [comment]
我最不習慣的應該是隱私問題！我是很需要自己的隱密空間的人（就是別人看不到我的那種），但宿舍除了廁所以外沒有這種空間。所以平常上課回宿舍之後都還是覺得累累的，沒有自己的小空間恢復能量的感覺...

**Comment by irenesunnn** [comment]
我覺得宿舍生活是我大學這幾年最精華的部分! 在有很合拍的室友的前提下，
不同背景的人在一起生活會有很多思想碰撞，也會無形中再次形塑你的想法或是眼界
後來室友都畢業，自己外宿之後，真的非常懷念宿舍生活!!!

**Comment by yue083139** [comment]
我覺得住宿舍的生活還算蠻有趣的，可以和來自不同地方的舍友們交流互動，有時快要期中了就會一起讀書，有時也會一起玩一起有些活動像是烤肉什麼的，還可以跟別人蹭uber one，缺點就是沒甚麼私人空間吧，不過我個人是沒這麼在意這點就是了。

**Comment by ed112601513** [comment]
我覺得住在家裡比較有個人空間和時間，但是生活會比住在宿舍枯燥乏味許多，而住宿舍會常常和室友互動，可能會有開心的事也可能會有衝突，這些都能成為自己難忘的經驗和回憶，在未來不管是人際關係的處理，或是在自我提升和修養，都能更加進步。

**Comment by yuan3x2** [comment]
我自己覺得住宿舍有利有弊，最大的差別大概就是在家有麻麻煮很多好吃的，但如果是有同系的話可以在期中考前夕進行抱團取暖環節，一起肝到天亮，我自己跟現在的室友住在一起兩年多了，有許多快樂的回憶，不孤單，所以我很珍惜～

**Comment by lai1008** [comment]
我覺得住宿舍是一個CP值很高的事，因為離教室很近而且住宿費也不高，但缺點就是不能想幹嘛就幹嘛像是把音樂撥出來之類的，如果自己住就不會有這些麻煩但缺點就是房租高而且離學校也會有點距離。

**Comment by Elaine4488** [comment]
我之前的室友每天固定12點就會睡覺，我們就會關大燈，所以有時候我也就跟著不會太晚睡。結果現在搬出來住，幾乎每天都熬夜，拖到超晚才睡，有時侯蠻想念那段作息比較正常的時光。

**Comment by 113607514** [comment]
住進宿舍已經快兩個月了，最大的不同是必須習慣與室友共享空間和處理所有日常生活的事務。相比家裡的舒適與隱私，宿舍的作息時間和環境有時會讓我感到不太適應，尤其是噪音和公共設施的使用上。儘管如此，這段時間也讓我學會了更多自理和與人相處的技巧，慢慢地開始習慣這樣的生活節奏。

**Comment by 113607514** [comment]
不習慣的應該就是和室友的睡覺時間不同。我以前覺得十二點就很晚了，沒想到室友們都會兩、三點才睡，結果現在我也變成十二點、一點多才睡的人，早點上床反而睡不著。室友還常開玩笑說：「你老人喔？那麼早睡。」不過他們人很好，每當看到我睡覺時，會特別放輕動作，讓我有點不好意思。有趣的是，經常會有人在兩、三點還大聲喧嘩，這時就會有人去敲門，甚至會在宿舍群組裡罵對方，對方也不甘示弱，回嗆說：「怕吵就不要住宿啊！」雖然我認為大家應該和諧相處，但其實看到這樣的互動，也覺得蠻有趣的，當然，我並不認同這樣的行為。

**Comment by Mminniehuang** [comment]
我覺得作息是最大的問題，不管是燈光還是稍微放輕的聲音（遑論有時候並沒有降低音量）都會影響到睡眠。其實也不只睡眠上的問題，宿舍讀書，旁邊卻不停的嘻笑吵鬧也是很大的困擾。
不過宿舍生活本來就是不斷的溝通和相互退讓，良好的室友關係往往建立在雙方友善尊重的態度！
